src/panels/lovelace/cards/energy/hui-energy-solar-graph-card.ts

```diff
@@ -455,7 +455,7 @@ export class HuiEnergySolarGraphCard
               } else {
                 dateObj.setMinutes(0, 0, 0);
               }
-              const time = dateObj.getTime();
+              const time = dateObj.getTime()-3.6e6;
```

Comment:
We should do the adjustment on line 446, so we calculate with the correct time from the start